importer/importer_utils.py

```diff
@@ -95,7 +95,7 @@ def remove_markup(text):
     remove_br = re.compile('<br.*?>\W*', re.I)
     text = remove_br.sub(' ', text)
     text = " ".join(text.split())
-    if "[" in text:
+    if "[" in text or "'''" in text or "''" in text:
```

Comment:
don't need to check for  `'''` as this is caught by `''`

Reply:
Removed.